internal/testplanet/planet.go

```diff
@@ -304,7 +304,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				Interval: 120 * time.Second,
 			},
 			Console: consoleweb.Config{
-				Address: "127.0.0.1:0",
+				Address: "127.0.0.1:8081",
```

Comment:
There can be multiple consoles and satellites, so this must not be hardcoded. Also this is likely to clash with a developer-s host:port (see https://github.com/storj/storj/wiki/Testing#automatically-choose-ports).

Reply:
Yes, I shouldn't commit this, Forgot to remove